src/components/InstrumentConfigForm.vue

```diff
@@ -75,6 +75,11 @@
       @input="update"
     />
     <!-- End of MUSCAT instrument specific fields -->
+    <b-alert variant="warning" :show="badFilterReadoutMode">
+      <strong>Warning:</strong> This filter/readout mode combination is not eligible for automatic reduction. <br><br>
+      Please contact science support if this
```

Comment:
Can we make "science support" into a link to `mailto:science-support@lco.global`?